geemap/toolbar.py

```diff
@@ -3505,27 +3500,12 @@ def toolbar_btn_click(change):
 
     toolbar_button.observe(toolbar_btn_click, "value")
 
-    def close_btn_click(change):
-        if change["new"]:
-            toolbar_button.value = False
-            if m is not None:
-                if m.tool_control is not None and m.tool_control in m.controls:
-                    m.remove_control(m.tool_control)
-                    m.tool_control = None
-                m.toolbar_reset()
-            toolbar_widget.close()
-
-            if hasattr(m, "_colorbar_ctrl") and (m._colorbar_ctrl is not None):
-                m.remove_control(m._colorbar_ctrl)
-                m._colorbar_ctrl = None
-
-    close_button.observe(close_btn_click, "value")
-
     toolbar_button.value = True
     if m is not None:
         toolbar_control = ipyleaflet.WidgetControl(
             widget=toolbar_widget, position="topright"
         )
+        setattr(toolbar_control, "cleanup", cleanup)
```

Comment:
Same here and some other place in the code.

Reply:
Done and below.